docs/articles/documentation/guides/continuous-integration/circleci.md

```diff
@@ -6,139 +6,139 @@ redirect_from:
-          keys:
-          - v1-dependencies-{{ checksum "package.json" }}
-          - v1-dependencies-
+      - node/install-packages
+      - run:
+          command: npm run test
+      - store_test_results:
+          path: /tmp/test-results
+workflows:
+  e2e-test:
+    jobs:
+      - test
+```
 
-      - run: yarn install
+The table below describes the YAML properties used in this example:
 
-      - save_cache:
-          paths:
-            - node_modules
-          key: v1-dependencies-{{ checksum "package.json" }}
+Property                | Description
+----------------------- | ----
+`version`               | The CircleCI version that should be used for this project.
+`orbs`                  | Imports CircleCI [orbs](https://circleci.com/docs/2.0/orb-intro/)–reusable parts of build configurations stored in a CircleCI registry.
+`node`                  | Imports the [circleci/node](https://circleci.com/developer/orbs/orb/circleci/node) orb v4.1.0 under the name `node`.
+`jobs`                  | The section that specifies [jobs](https://circleci.com/docs/2.0/jobs-steps/#jobs-overview).
+`test`                  | The job's name.
+`docker`                | An [executor](https://circleci.com/docs/2.0/configuration-reference/#docker--machine--macos--windows-executor) that defines that defines the environment in which this job runs. The `docker` executor specifies that the job should run inside a docker container.
+`image`                 | Specifies the docker image to use. `circleci/node:latest-browsers` is a pre-built CircleCI docker image with the latest version of node and browsers pre-installed. For the full list of available images, see [Pre-Built CircleCI Docker Images](https://circleci.com/docs/2.0/circleci-images/#nodejs).
+`steps`                 | The job's [steps](https://circleci.com/docs/2.0/jobs-steps/#steps-overview).
+`checkout`              | Checks out code from the `testcafe-ci-demo` repository. This is a pre-defined step (see [checkout](https://circleci.com/docs/2.0/configuration-reference/#checkout)).
+`node/install-packages` | A step from the `node` orb that installs the project's dependencies (see [install-packages](https://circleci.com/developer/orbs/orb/circleci/node#commands-install-packages)).
+`run`                   | The [run](https://circleci.com/docs/2.0/configuration-reference/#run) step invokes shell commands. In this example, `run` executes the `npm test` script.
+`store_test_results`    | Uploads test results to the specified directory (see [store_test_results](https://circleci.com/docs/2.0/configuration-reference/#store_test_results)).
+`workflows`             | A [workflow](https://circleci.com/docs/2.0/workflows/) that runs the `test` job.
 
-      - run: yarn test
+After you pasted the YAML, click **Add Config**.
 
-      - store_test_results:
-          path: /tmp/test-results
-{% endraw %}
-```
+![Click Add Config to save the configuration](../../../images/circle-ci/circleci-config.png)
```

Comment:
This screenshot shows a different config, need to re-capture

Reply:
Recaptured